app/controllers/votes_controller.rb

```diff
@@ -12,7 +12,8 @@ def create
     @vote = @idea.votes.build
     @vote.user = current_user
     @vote.save
-    redirect_to @idea, notice: 'Your vote has been added'
+    # redirect_to @idea, notice: 'Your vote has been added'
```

Comment:
Is this code intentionally commented?

Reply:
This needs a ruby dev to look at these controller actions to make them work with and without the JS.